pyproject.toml

```
@@ -36,13 +36,13 @@ whattimeisit-provider = "examples.flask.whattimeisitrightnow.app.app:main"
 
 [tool.poetry.dependencies]
 python = ">=3.8"
-globus-sdk="^3.9.0"
-jsonschema = "^4.17"
-pyyaml = "^6"
-pydantic = "^1.7.3"
-isodate = "^0.6.0"
-cachetools = "^5.0"
-flask = {version = "^2.3.0", optional = true}
+globus-sdk=">=3.9,<4"
+jsonschema = ">=4.17,<5"
+pyyaml = ">=6,<7"
+pydantic = ">=1.7.3,<2"
```

Technically, the exact translation should be:

```suggestion
pydantic = ">=1.7.3,<1.8"
```

and the Globus SDK and Flask versions should be similarly restricted, but I'm not strongly concerned.

---

I thought so as well, but when I dug into the built packages, this is part of what I found in the sdist:
```
Requires-Dist: globus-sdk (>=3.9.0,<4.0.0)
Requires-Dist: isodate (>=0.6.0,<0.7.0)
Requires-Dist: jsonschema (>=4.17,<5.0)
Requires-Dist: pydantic (>=1.7.3,<2.0.0)
```

I realized that when we last spoke, we attributed to `^` the rules which go with `~`.
Docs: https://python-poetry.org/docs/dependency-specification/#caret-requirements

So... I just want these non-standard things which can even _lead to this conversation_ to go away. :grin: